src/core/stream/frame.mli

```diff
@@ -26,42 +26,45 @@
 
 type field
 
+val string_of_field : field -> string
+val field_of_string : string -> field
+val register_field : string -> field
+
 module Fields : Map.S with type key = field
 
+(* Default fields *)
 val audio_field : field
 val video_field : field
 val midi_field : field
+val audio_field_n : int -> field
+val video_field_n : int -> field
+
+(* Find fields *)
 
 (* The following raise [Not_found] when the field does not exist. *)
```

Comment:
Outdated comment:
```suggestion
```
As a general remark, it's nice to move away from `Not_found` exceptions and use options instead!

Reply:
Agreed!